lib/kadai-core/src/main/java/io/kadai/task/internal/TaskServiceImpl.java

```diff
@@ -963,6 +968,7 @@ public BulkOperationResults<String, KadaiException> forceCompleteTasks(List<Stri
   }
 
   @Override
+  @Deprecated(forRemoval = true, since = "12.0.0")
```

Comment:
```suggestion
  @Deprecated(forRemoval = true, since = "11.0.0")
```

I told you wrong. This is right now :)